kili/mutations/asset/queries.py

```diff
@@ -2,39 +2,9 @@
 Queries of asset mutations
 """
 
+from kili.graphql.operations.asset.fragments import ASSET_FRAGMENT
```

Comment:
should be moved to operations/asset/fragments.py ?

Reply:
you mean the PROJECT_FRAGMENT_ID ?